pkg/pillar/cmd/baseosmgr/handlebaseos.go

```diff
@@ -295,12 +299,15 @@ func doBaseOsActivate(ctx *baseOsMgrContext, uuidStr string,
 			publishBaseOsStatus(ctx, status)
 			return changed
 		}
+		zboot.SetOtherPartitionStateUpdating(log)
 		// move the state from VERIFIED to INSTALLED
 		setProgressDone(status, types.INSTALLED)
 		publishZbootPartitionStatus(ctx, status.PartitionLabel)
 		baseOsSetPartitionInfoInStatus(ctx, status,
 			status.PartitionLabel)
 		publishBaseOsStatus(ctx, status)
+	} else {
+		return changed
```

Comment:
Shouldn't we log something here like "waiting for XXX"?
Note that Avi's PR will have more reasons for !done

Reply:
Resolved